packages/adapter-nuxt/src/hooks/documentation-read.ts

```diff
@@ -59,20 +60,21 @@ export const documentationRead: DocumentationReadHook<PluginOptions> = async (
 		} else {
 			fileContent = stripIndent`
 				<script ${scriptAttributes.join(" ")}>
-					import { components } from "~/slices";
-
-					const prismic = usePrismic();
-					const { data: page } = useAsyncData("[${model.id}]", () =>
-						prismic.client.getSingle("${model.id}")
-					);
-
-					useHead({
-						title: page.value?.data.meta_title,
-						meta: [{
-							name: "description",
-							content: page.value?.data.meta_description,
-						}],
-					});
+				import { components } from "~/slices";
```

Comment:
❓ You removed the indentation here, is that wanted?

For example, `<template>` still has the same indentation.

Reply:
Yes, that was an error